sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/MapElementsTest.java

```diff
@@ -166,12 +231,31 @@ public Integer apply(KV<Integer, String> input) {
   }
 
   /**
-   * Basic test of {@link MapElements} with a {@link SerializableFunction}. This style is generally
-   * discouraged in Java 7, in favor of {@link SimpleFunction}.
+   * Test of {@link MapElements} coder propagation with a parametric {@link InferableFunction} where
+   * the type variable occurs nested within other concrete type constructors.
```

Comment:
This looks out of place in the diff. The removed lines are the docstring for `testMapBasicSerializableFunction` which has been renamed to `testMapBasicProcessFunction` and appears as the next function below.